### PR TITLE
Update for Elgg 3.3

### PR DESCRIPTION
**Pull request by rohit1290**
Code update for elgg 3.3 and some minor code cleanup

### CODE REVIEW

File: start.php [hunk 1]
```diff
@@ -1,20 +0,0 @@
-<?php
-/**
- * Main plugin file
- *
- */
-
-require_once __DIR__ . '/lib/functions.php';
```
Comment (jdalsem) [inline start.php:7]: where did this go?

**Review by jdalsem — CHANGES_REQUESTED** [review]

File: start.php (hunk 1 above)
Comment (jdalsem) [reply start.php:7]: maybe include it in the elgg-plugin.php

Comment (rohit1290) [reply start.php:7]: Done

**Comment by jdalsem** [comment]
Thanks

**Pull request merged**